### DISCUSSION TITLE
Recur monthly?

### DISCUSSION THREAD

**Discussion by xpseudonym** [Extended-Thunder/send-later#571] [Q&A] [answered]
Hi Folks,
The 'Recur monthly' function seems to offer only recurring on a particular day of the month - i.e. 1st Sunday; 2nd Monday etc.  How to I set it to recur on the 11th of every month (what ever day of the week that happens to  be)?
Many thanks.
Morgan.
PS And, how do I buy send latter a coffee?  (I'm sure that's bleeding obvious somewhere too...)

**Comment by jikamens** [comment, marked as answer]
>The 'Recur monthly' function seems to offer only recurring on a particular day of the month - i.e. 1st Sunday; 2nd Monday etc. How to I set it to recur on the 11th of every month (what ever day of the week that happens to be)?

Set the date of the first occurrence, and then select "monthly" next to "Recur" in the pop-up, and it will behave exactly how you described.

>PS And, how do I buy send latter a coffee? (I'm sure that's bleeding obvious somewhere too...)

Thank you for asking!

It's _not_ obvious, because for a while (like, until about a half hour ago :wink:) we didn't have donations set up. We do now: you can donate to Extended Thunder Inc., the non-profit charitable organization which supports Send Later development and maintenance via either [Liberapay](https://liberapay.com/ExtendedThunder) for recurring donations or [PayPal](https://www.paypal.com/paypalme/ExtendedThunder) for single donations. If you are in the United States, your donations are tax-deductible to the extent permitted by law.
